package.json

```diff
@@ -18,7 +18,14 @@
   },
   "homepage": "https://www.figma.com/design/6bg5Y3Y4gLXgSdVa8Da8Vy/Wellcome-design-system?node-id=316-9&t=c6cHC23OsadCwIg3-0",
   "devDependencies": {
+    "@svgr/core": "^8.1.0",
     "@tokens-studio/sd-transforms": "^0.16.1",
     "style-dictionary": "^4.0.0-prerelease.27"
+  },
+  "dependencies": {
+    "@svgr/plugin-jsx": "^8.1.0",
+    "@svgr/plugin-prettier": "^8.1.0",
+    "@svgr/plugin-svgo": "^8.1.0",
```

Comment:
Should we install all of the `@svgr` packages as dev dependencies?

Reply:
probably yes 👍